packages/utils/README.md

```diff
@@ -0,0 +1,64 @@
+# Form5 / Utils
+
+## Demo
+
+[https://form5.github.io/](https://form5.github.io/)
```

Comment:
If you want to have this kind of link you I'll need to put demo on `form5/from5.github.io`.
If you put source code of this link will be `https://form5.github.io/form5`

https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site